indexer/services/ender/src/handlers/update-clob-pair-handler.ts

```diff
@@ -18,6 +26,40 @@ export class UpdateClobPairHandler extends Handler<UpdateClobPairEventV1> {
 
   // eslint-disable-next-line @typescript-eslint/require-await
   public async internalHandle(): Promise<ConsolidatedKafkaEvent[]> {
+    if (config.USE_UPDATE_CLOB_PAIR_HANDLER_SQL_FUNCTION) {
+      return this.handleViaSqlFunction();
+    }
+    return this.handleViaKnex();
+  }
+
+  private async handleViaSqlFunction(): Promise<ConsolidatedKafkaEvent[]> {
+    const eventDataBinary: Uint8Array = this.indexerTendermintEvent.dataBytes;
+    const result: pg.QueryResult = await storeHelpers.rawQuery(
+      `SELECT dydx_update_clob_pair_handler(
+        '${JSON.stringify(UpdateClobPairEventV1.decode(eventDataBinary))}'
+      ) AS result;`,
+      { txId: this.txId },
+    ).catch((error: Error) => {
+      logger.error({
+        at: 'UpdateClobPairHandler#handleViaSqlFunction',
+        message: 'Failed to handle UpdateClobPairEventV1',
+        error,
+      });
+
+      throw error;
+    });
+
+    const perpetualMarket: PerpetualMarketFromDatabase = PerpetualMarketModel.fromJson(
+      result.rows[0].result.perpetual_market) as PerpetualMarketFromDatabase;
+
```

Comment:
I think we need to update the `perpetualMarketRefresher` in-memory cache here? Looks like we never added tests for this in the knex handler.

Reply:
Expanded the test and fixed the code.